backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -235,13 +268,22 @@ public long getPatientsCount(
       UUID facilityId,
       ArchivedStatus archivedStatus,
       String namePrefixMatch,
-      boolean includeArchivedFacilities) {
+      boolean includeArchivedFacilities,
+      String orgExternalId) {
     if (namePrefixMatch != null && namePrefixMatch.trim().length() < MINIMUM_CHAR_FOR_SEARCH) {
       return 0;
     }
+
+    Organization orgToSearch;
+    if (orgExternalId != null && StringUtils.isNotEmpty(orgExternalId)) {
```

Comment:
[StringUtils.IsNotEmpty](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#:~:text=to%20isNotBlank(CharSequence)-,isNotEmpty,-public%20static%C2%A0boolean) actually does a null check as well so you can get rid of your null check!

Reply:
OOO! that's good to know! I'll fix these up! Thank you!